pkgs/by-name/cx/cxt/package.nix

```
@@ -0,0 +1,29 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cxt";
+  version = "0.1.0";
```

```suggestion
  version = "0.1.1";
```
is the most up to date version